src/containers/contactsPage/ContactsPage.js

```diff
@@ -1,48 +1,70 @@
-import React, {useState, useEffect} from "react";
-import {ContactForm} from '../../components/contactForm';
-import {TileList} from '../../components/tileList'
+import React, { useState, useEffect } from "react";
+import { ContactForm } from "../../components/contactForm/ContactForm";
+import { TileList } from "../../components/tileList/TileList";
 
-export const ContactsPage = ({contactsArray, addContact}) => {
+export const ContactsPage = ({ contactsArray, addContact }) => {
   /*
   Define state variables for 
   contact info and duplicate check
   */
-const [name, setName] = useState('');
-const [phoneNumber, setPhoneNumber] = useState('');
-const [email, setEmail] = useState('');
-const [duplicateName, setDuplicateName] = useState(false);
+  const [name, setName] = useState("");
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [email, setEmail] = useState("");
+  const [isDuplicate, setIsDupliate] = useState(false);
 
   const handleSubmit = (e) => {
     e.preventDefault();
     /*
     Add contact info and clear data
     if the contact name is not a duplicate
     */
+    if (isDuplicate) {
+      setName("");
+      return;
+    } else {
+      addContact(name, phoneNumber, email);
+      setName("");
+      setPhoneNumber("");
+      setEmail("");
+    }
   };
 
   /*
   Using hooks, check for contact name in the 
   contacts array variable in props
   */
-
+  console.log(isDuplicate);
   useEffect(() => {
-    const findDuplicate = (contactsArray, duplicateName) => {
-      const namePropertyInArray = contactsArray[name]
-      if (found) {
-        alert('Name is already in contacts');
+    console.log(contactsArray);
```

Comment:
Don't forget to remove this. Always good to have a "clean" PR as a habit. As if you were publishing to a production system.